buildSrc/src/main/java/org/elasticsearch/gradle/info/GlobalBuildInfoPlugin.java

```diff
@@ -95,6 +97,7 @@ public void apply(Project project) {
             ext.set("minimumRuntimeVersion", minimumRuntimeVersion);
             ext.set("gradleJavaVersion", Jvm.current().getJavaVersion());
             ext.set("gitRevision", gitRevision);
```

Comment:
Since we are setting the git revision here. We can remove it completely from the generate and print tasks.

Reply:
Let's leave changes to the handling of the git revision to a separate pull request. Is the date handling here fine now?